services/admin/src/store.test.ts

```diff
@@ -412,6 +415,99 @@ test('add a CVR file with mixed live and test CVRs', async () => {
   });
 });
 
+test('add a CVR file with an invalid election id', async () => {
+  const { electionDefinition } = electionMinimalExhaustiveSampleFixtures;
+
+  const store = Store.memoryStore();
+  store.addElection(electionDefinition.electionData);
+
+  const { cvrFile } = primaryElectionSampleFixtures;
+
+  const result = await store.addCastVoteRecordFile({
+    electionId: 'not-a-real-election-id',
+    filePath: cvrFile.asFilePath(),
+  });
+
+  expect(result.isErr()).toBe(true);
```

Comment:
Maybe not necessary? The following line would fail whenever this would fail, and probably with a better error message.

Reply:
ah, true - blind copy-paste from old tests 😅